packages/babel-helpers/src/helpers.js

```diff
@@ -372,6 +372,29 @@ helpers.extends = defineHelper(`
   }
 `);
 
+helpers.extendsDefine = defineHelper(`
```

Comment:
Maybe we could call this `extendSpread`? Or `mergeSpread`

Reply:
What about `spreadByDefine` to keep the define idea?

Reply:
@nicolo-ribaudo is `spreadByDefine` ok for you?

Reply:
Yes